### ISSUE TITLE
get_all_related_objects is an unofficial API that has been deprecated

### ISSUE THREAD

**Issue by alexahn** [scoursen/django-softdelete#28]
```
RemovedInDjango110Warning: 'get_all_related_objects is an unofficial API that has been deprecated. You may be able to replace it with 'get_fields()'
for x in self._meta.get_all_related_objects():
```
I'll submit a pull request in the near future regarding this, just a heads up.

**Comment by bdupharm** [comment]
+1

**Comment by scoursen** [comment]
This should be addressed with commit 3509a8470d55d811997ebae11219e637ffa32293
